tracpro/templates/frame.html

```diff
@@ -29,6 +29,7 @@
       <![endif]-->
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <script type="text/javascript" src="{{ STATIC_URL }}js/jqcloud-1.0.4.min.js"></script>
```

Comment:
I'm not sure why exactly but for some reason the word cloud would not work until I moved it.